revocation.md

```diff
@@ -0,0 +1,237 @@
+# Revocation
+
+**Patent Pending**
+
+Revocation requires that a verifiable revocation notice for a given Doughnut be posted somewhere
+accessible to all Doughnut revokers and verifiers.
+
+Because Doughnuts support decentralisation it makes sense for a Doughnut to tell verifiers
+how and where to revoke it, and where to check its revocation status.
+This removes the need for centralised revocation services.
+
+What follows are specifications for decentralised Doughnut revocation methods.
+A Doughnut version may specify a specific revocation method, or allow the issuer to
+make a choice of revocation method.
+
+Different strategies for achieving revocation are explored in the [`Strategies`](#strategies) section below.
+
+
+
+
+
+# Methods
+
+
+
+## 0
+Method 0 specifies a basic HTTP revocation interface with issuer/holder revocation.
```

Comment:
We may want not to descend on anything lower secure than HTTPS for this service.